model/DataQuery.php

```diff
@@ -13,6 +13,8 @@
  */
 class DataQuery {
 
+	public $lastAlias;
```

Comment:
Did you look into storing the alias with `$this->setQueryParam()`? Would mean we don’t need to introduce new things to the public API